### DISCUSSION TITLE
Option for unlimited list length

### DISCUSSION THREAD

**Discussion by ghost** [umbrellaplug/umbrellaplug.github.io#180] [Ideas]
New on board with Umbrella following the trakt outage making Seren unusable. Loving it. For me, there is only one issue that I have (and it's small). I recognize that this will slow down the list creation and make entering a long list a slow process, however, this is a sacrifice that I'd like to be able to make as a choice.

Presently there is a maximum number of items in a list that produces pagination. For me, this is not as seamless as just having all of them listed. I know this will produce a slow app but I would be happy to make that choice for myself. Perhaps a tickbox to remove pagination with a clear "Here be dragons!" warning?

A nice to have, for me.

**Comment by umbrellaplug** [comment]
Welcome to the party. Pagination already goes to 60. How high are you wanting it?

Before you answer unlimited, I'm not sure that's an option with the trakt api. I can check, but what's a realistic number?

You're gonna blow your device up. 😀

**Reply by ghost** [reply]
I have about 300 in my movies and they all load at once in Seren. Have a Vero 4k+ so I reckon I can use it as a heater if it fails 5555

If you can show me what to edit I can do this locally and avoid people getting upset with you :)

**Reply by umbrellaplug** [reply]
Hey Was looking around and noticed there's a setting in Trakt section for "Paginate lists when possible". Have you tried turning that off?

**Comment by umbrellaplug** [comment]
Kodi\userdata\addon_data\plugin.video.umbrella\settings.xml
`<setting id="page.item.limit" default="true">20</setting>`

Set that 20 to like 1000

**Comment by ghost** [comment]
Aha, no I haven't. Will look for that this evening.
